general/CatalogoDatiInformativiPerPsP.xsd

```diff
@@ -0,0 +1,552 @@
+        <xs:annotation>
+          <xs:documentation>
+						identificativo del PSP
+					</xs:documentation>
+        </xs:annotation>
+      </xs:element>
+      <xs:element type="stText70" name="ragioneSociale">
+        <xs:annotation>
+          <xs:documentation>
+						Ragione Sociale del PSP
+					</xs:documentation>
+        </xs:annotation>
+      </xs:element>
+      <xs:element type="stAbiCode" name="codiceABI" minOccurs="0">
+        <xs:annotation>
+          <xs:documentation>
+						Identificativo del PSP all'interno dell'Associazione Bancaria Italiana 
+					</xs:documentation>
+        </xs:annotation>
+      </xs:element>
+      <xs:element type="stBicCode" name="codiceBIC" minOccurs="0">
+        <xs:annotation>
+          <xs:documentation>
+						codice indetificativo internazionale
+					</xs:documentation>
+        </xs:annotation>
+      </xs:element>
+
+      <xs:element type="stMyBankIdVs" name="mybankIDVS" minOccurs="0">
+        <xs:annotation>
+          <xs:documentation>
+						identificativo del PSP all'interno del servizio MyBank. 
+						Dato necessario qualora si disponga di un canale di tipo MYBK
+					</xs:documentation>
+        </xs:annotation>
+      </xs:element>
+
+      <xs:element type="ctInformativaMaster" name="informativaMaster">
+        <xs:annotation>
+          <xs:documentation>
+						Tale campo descrive informazioni generali
+					</xs:documentation>
+        </xs:annotation>
+      </xs:element>
+
+      <xs:element type="ctListaInformativaDetail" name="listaInformativaDetail">
+        <xs:annotation>
+          <xs:documentation>
+						Configurazione dei servizi di pagamento
+					</xs:documentation>
+        </xs:annotation>
+      </xs:element>
+    </xs:sequence>
+  </xs:complexType>
+
+  <xs:complexType name="ctListaInformativaDetail">
+    <xs:annotation>
+      <xs:documentation>
+				Descrizione dettagliata dei singoli servizi di pagamento offerti.
+				i valori di default sono quelli da utilizzare con intermediario CBILL
+			</xs:documentation>
+    </xs:annotation>
+    <xs:sequence>
+      <xs:element type="ctInformativaDetail" name="informativaDetail" maxOccurs="unbounded" minOccurs="1">
+        <xs:annotation>
+          <xs:documentation>Configurazione dei canali</xs:documentation>
+        </xs:annotation>
+      </xs:element>
+    </xs:sequence>
+
+  </xs:complexType>
+
+  <xs:complexType name="ctListaInformazioniServizio">
+    <xs:sequence>
+      <xs:element type="ctInformazioniServizio" name="informazioniServizio" maxOccurs="unbounded" minOccurs="1" />
+    </xs:sequence>
+  </xs:complexType>
+
+
+  <xs:complexType name="ctListaParoleChiave">
+    <xs:sequence>
+      <xs:element type="stParoleChiave" name="paroleChiave" maxOccurs="5" minOccurs="1" />
+    </xs:sequence>
+  </xs:complexType>
+
+  <xs:complexType name="ctListaFasceCostoServizio">
+    <xs:sequence>
+      <xs:element type="ctFasciaCostoServizio" name="fasciaCostoServizio" maxOccurs="8" minOccurs="1" />
```

Comment:
Il vincolo "maxOccurs=8" era stato rilassato per le convenzioni (un PSP potrebbe avere diverse convenzioni con diverse PA). Introducendo nuovamente il vincolo a 8, si limita questa funzionalità (non credo sia un problema, non mi pare sia molto utilizzata, ma non so quali indicazioni siano state date all'epoca ai PSP).

Reply:
Resta invariato per l'XSD verso le PA, è stato introdotto solo versi i PSP @gammam ai PSP è stata/verrà data comunicazione in merito corretto ?